src/Components/index.js

```diff
@@ -0,0 +1,9 @@
+import { Column } from '../Components/Column/Column'
+import { Home } from '../Routes/Home/Home'
+import { Deposit } from '../Routes/Deposit/Deposit'
+import { Withdraw } from '../Routes/Withdraw/Withdraw'
+import { Balance } from '../Routes/Balance/Balance'
+import { NavigateHomePage } from '../Components/NavigateHomePage/NavigateHomePage'
```

Comment:
Remove pages from here ..they are not components.

Reply:
done